tests/snapshots/tests.contrib.wsgi.test_wsgi.test_distributed_tracing_nested.json

```diff
@@ -10,23 +10,24 @@
     "error": 0,
     "meta": {
       "_dd.base_service": "",
-      "_dd.p.tid": "654a694400000000",
```

Comment:
We're no longer `_dd.p.tid` on some spans. Is this expected?

Reply:
No, but my testing shows that it's orthogonal to what's being tested here. I've backed out this change anyway.